src/webgpu/shader/validation/expression/call/builtin/subgroupShuffle.spec.ts

```diff
@@ -94,6 +94,54 @@ g.test('early_eval')
     t.expectCompileResult(t.params.stage === 'runtime', code);
   });
 
+g.test('param2_early_eval')
+  .desc('Ensures delta/mask parameters must be in the range [0, 128) for const and override')
+  .params(u =>
+    u
+      .combine('op', kOps)
+      .combine('value', [0, 127, 128] as const)
```

Comment:
I think subgroupShuffle (not up/down/xor variants) can have a signed value.

Reply:
Good catch, I'll update.